tekton-pipelines/images/artcd/Dockerfile.base

```diff
@@ -0,0 +1,36 @@
+FROM registry.access.redhat.com/ubi9/ubi:latest AS builder
```

Comment:
Could the file be Containerfile.extension? Let's do our tiny bit to work towards a new standard :) https://github.com/containers/buildah/discussions/3170